### DISCUSSION TITLE
NextCloud showing blank page

### DISCUSSION THREAD

**Discussion by scrummie02** [nextcloud/all-in-one#3762] [Questions] [answered]
### Steps to reproduce
1. bring up nextcloud in browser
2. blank page

### Actual behavior blank page
![Screenshot 2023-11-16 113155](https://github.com/nextcloud/all-in-one/assets/572687/c1ca057c-6ebf-44d3-b052-d4a6e4622472)

### Host OS 
Debian and docker with AIO

**Comment by szaimen** [comment]
Hi, are you using Cloudflare and their Rocket Loader feature?

**Comment by scrummie02** [comment]
No.  No CF

**Comment by szaimen** [comment, marked as answer]
Can you check if it works from a different network/device/browser?
